client/reader/discover/helper.js

```diff
@@ -65,88 +61,6 @@ export function getTagsFromStreamKey( streamKey = '' ) {
 	return [];
 }
 
-function hasDiscoverSlug( post, searchSlug ) {
-	const metaData = get( post, 'discover_metadata.discover_fp_post_formats' );
-	return !! ( metaData && find( metaData, { slug: searchSlug } ) );
-}
-
-export const discoverBlogId = config( 'discover_blog_id' );
-
-export function isDiscoverBlog( blogId ) {
-	return +blogId === config( 'discover_blog_id' );
-}
-
-export function isDiscoverFeed( feedId ) {
-	return +feedId === config( 'discover_feed_id' );
-}
-
 export function isDiscoverEnabled() {
 	return true;
```

Comment:
It feels like this helper could also be removed since it's always `true`.

Reply:
Good call - removed in [35867b2](https://github.com/Automattic/wp-calypso/pull/80263/commits/35867b26996996586a9f1fb04466503e539dbb7e).